solution/CSV_Quiz.ipynb

```diff
@@ -0,0 +1,63 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "3b5c1ad8",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "my name is chravin.\n",
+      "my father name is jayeshbai.\n",
+      "my mother name is ritaben.\n",
+      "i am 18 years old.\n",
+      "my brother name is nol.\n",
+      "my grandfather name is chanubhai.\n",
+      "my grandmnother name is chandanben.\n",
+      "my sister name is dhruvi.\n",
+      "i am living at kani.\n",
+      "i am study in vtcbcsr in bca.\n",
+      "i am bca 3rd sem student.\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(\"quiz.txt\",'r') as file:\n",
+    "    rec=file.read()\n",
+    "    print(rec)"
```

Comment:
another way to print record?

Reply:
Using For Loop.

Reply:
can you please print record using for loop?

Reply:
file_name = open("quiz.txt",'r')
for lines in file_name:
    print(lines)

This way the To Print The Record Using The For loop